src/main/java/com/jcraft/jsch/OpenSSHConfig.java

```diff
@@ -303,6 +312,7 @@ public int getPort() {
         port = Integer.parseInt(foo);
       } catch (NumberFormatException e) {
         // wrong format
```

Comment:
I think we can drop this `// wrong format` comment here: I believe I added it because some static analysis tool didn't like any empty catch statement, but that issue is now solved by the addition of `logError()` call.

Reply:
sure